lighthouse-core/gather/driver.js

```diff
@@ -571,6 +571,17 @@ class Driver {
   }
 
   /**
+   * Return the body of the response with the given ID.
+   * @param {string} requestId
+   * @return {string}
+   */
+  getRequestContent(requestId) {
+    return this.sendCommand('Network.getResponseBody', {
+      requestId,
+    }).then(result => result.body);
```

Comment:
add a comment in here that we're ignoring `base64Encoded` ?

// ignoring the result.base64Encoded boolean, which indicates if body is already encoded